Vagrantfile

```diff
@@ -43,14 +45,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         echo "Installing required packages"
         set -x
         sudo apt-get update
-        sudo apt-get install --yes git python3-pip python3-venv rabbitmq-server amqp-tools mosquitto mosquitto-clients httpie socat
+		sudo apt-get upgrade
+        sudo apt-get install --yes git python3-pip python3-venv rabbitmq-server amqp-tools mosquitto mosquitto-clients httpie socat mariadb-server asterisk ffmpeg
 
         # Git settings for `root`.
         sudo git config --global pull.ff only
 
         # Git settings for `vagrant`.
         su vagrant -c "git config --global pull.ff only"
     SHELL
+	
+	# Setup MariaDB
+    machine.vm.provision :shell, privileged: true, inline: <<-SHELL
+        debconf-set-selections <<< "mysql-server mysql-server/root_password password secret"
+		debconf-set-selections <<< "mysql-server mysql-server/root_password_again password secret"
+		#apt-get -y install mysql-server
+		mysql -e "CREATE DATABASE secpi;"
```

Comment:
Can we stick to this name, please?
```suggestion
		mysql -e "CREATE DATABASE `secpi-testdrive`;"
```

Reply:
Sure. Sorry!

Reply:
Oh in the case of Vagrant, it would be `secpi-development`, no?

Reply:
Hm, no. We want to run the test suite on it, and it will use _that_ name. So, let's also stick with `secpi-testdrive`. Or create _both_ databases, `secpi-testdrive` _and_ `secpi-development`?

Reply:
With cda0f663, the `Vagrantfile` is now creating both databases.